web/package.json

```diff
@@ -34,7 +34,8 @@
     "test": "craco test",
     "eject": "craco eject",
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "fix": "eslint --fix src/*.js src/backend/*.js"
+    "fix": "eslint --fix src/*.js src/backend/*.js",
```

Comment:
CI checker means this: https://github.com/casdoor/casdoor/pull/1218

![image](https://user-images.githubusercontent.com/3787410/197379350-a78ea8eb-1259-4e85-9507-217baa76869c.png)

I didn't see you modified the CI code